### PR TITLE
[backend/frontend] fix date filters with same name in widgets (#7350)

### LINKED ISSUE #7350: [Dashboard] Multiple filters with the same name
## Description

When I want to filter on "Start date", several filters appear without me understanding why. Are these duplicates? Are these different filters but with the same name?

## Environment

OCTI 6.1.10

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Create a widget list
2. Use the knowledge perspective
3. Look for "start date" filter in "Dynamic source/target filters"
-> Several appear

## Additional information

By the way, I don't know if it's a bug but I don't have a "stop date" filter

## Screenshots 

Screenshot 2024-06-11 183539

### PR DESCRIPTION
**Pull request by Archidoit**
issue: https://github.com/OpenCTI-Platform/opencti/issues/7350

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ const stixDomainObjectsAttributes: { [k: string]: Array<AttributeDefinition> } =
     { name: 'objective', label: 'Objective', type: 'string', format: 'short', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: true },
   ],
   [ENTITY_TYPE_CONTAINER_NOTE]: [
-    { ...created, mandatoryType: 'external', editDefault: true, label: 'Publication date' },
```
Comment (labo-flg) [inline opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts:98]: I'm unsure of this modification. It was a mistake from the start ? I just want to be sure it was not intended to have a specific label for `created` in Notes.

**Comment by labo-flg** [comment]
Tested ok :
image
image

This one "Date" might be more precise too:
image

File: opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts (hunk 1 above)
Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts:98]: Yes the label was different, but checked with @RomainGUIGNARD to uniformize the created attribute translation with with the other entity types

Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts:98]: What's more, the publication date filter is available for Notes, but Note overview it is displayed as 'original creation date'

**Comment by Archidoit** [comment]
done for 'Date' filter @labo-flg

**Review by labo-flg — APPROVED** [review]

**Pull request merged**